packages/renderer/src/lib/kube/KubeActions.svelte

```diff
@@ -1,5 +1,11 @@
 <script lang="ts">
 import KubeApplyYamlButton from '/@/lib/kube/KubeApplyYAMLButton.svelte';
+
+interface Props {
+  disabled?: boolean;
```

Comment:
There could be several actions, and maybe not all of them are disabled when the resource is not accessible. Seems like this should just be an `accessible` or permissions property and below we decide which action(s) it disables.